src/sentry/utils/pytest/kafka.py

```diff
@@ -35,8 +35,12 @@ def delete_topic(self, topic_name):
         try:
             futures_dict = self.admin_client.delete_topics([topic_name])
             self._sync_wait_on_result(futures_dict)
-        except Exception:  # noqa
+        except Exception as e:  # noqa
```

Comment:
KafkaError{code=UNKNOWN_TOPIC_OR_PART,val=3,str="Broker: Unknown topic or partition"}

ingest-events

Reply:
hmm ```[2021-11-16 05:22:02,222] WARN Exception causing close of session 0x0: ZooKeeperServer not running (org.apache.zookeeper.server.NIOServerCnxn)```

Reply:
Oh weird I think this is a timing issue...